bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -130,3 +130,91 @@ Equipment,Oven,WhiteGood,Oven,Ovens,
+Equipment,GlassBreakDetector,Sensor,Glass Break Detector,,
+Equipment,LeakSensor,Sensor,Leak Sensor,,
+Equipment,IlluminanceSensor,Sensor,Illuminance Sensor,,
+Equipment,WaterMeter,Sensor,Water Meter,,
+Equipment,GasMeter,Sensor,Gas Meter,,
+Equipment,ElectricMeter,Sensor,Electric Meter,,
+Equipment,AirQualitySensor,Sensor,Air Quality Sensor,,
+Equipment,WaterQualitySensor,Sensor,Water Quality Sensor,,
+Equipment,HumiditySensor,Sensor,Humidity Sensor,Hygrometer,
+Equipment,WeatherStation,Sensor,Weather Station,,
+Equipment,TemperatureSensor,Sensor,Temperature Sensor,,
+Equipment,HeatDetector,Sensor,Heat Detector,Flame detector,
+Equipment,OccupancySensor,Sensor,Occupancy Sensor,,
+Equipment,ControlDevice,,Control Device,,
+Equipment,Keypad,ControlDevice,Keypad,,
+Equipment,Button,ControlDevice,Button,,
+Equipment,Slider,ControlDevice,Slider,,
+Equipment,Dial,ControlDevice,Dial,Rotary dial,
+Equipment,WindowCovering,,Window Covering,,
+Equipment,Drapes,WindowCovering,Drapes,Curtains,
+Equipment,Microwave,WhiteGood,Microwave,,
+Equipment,Cooktop,WhiteGood,Cooktop,,
+Equipment,Range,WhiteGood,Range,,
+Equipment,IceMaker,WhiteGood,Ice Maker,,
+Equipment,Toaster,WhiteGood,Toaster,Toaster Oven,
+Equipment,Fryer,WhiteGood,Fryer,Air fryer,
+Equipment,FoodProcessor,WhiteGood,Food Processor,,
+Equipment,Mixer,WhiteGood,Mixer,,
+Equipment,ExhaustFan,Fan,Exhaust Fan,"Extract Fan, Toilet Fan",
+Equipment,PowerSupply,,Power Supply,,
+Equipment,UPS,PowerSupply,UPS,Uninterrupted power supply,
+Equipment,Generator,PowerSupply,Generator,Emergency generator,
+Equipment,SolarPanel,PowerSupply,Solar Panel,,
+Equipment,EVCharger,PowerSupply,EV Charger,"EVSE, Electric Vehicle Supply Equipment, EV Charger, Car Charger, Charge Point, Charging Station",
+Equipment,WindGenerator,PowerSupply,Wind Generator,Wind Mill,
+Equipment,TransferSwitch,PowerSupply,Transfer Switch,,
+Equipment,Downlight,LightSource,Downlight,"Can Light, Pot Light ",
+Equipment,TrackLight,LightSource,Track Light,,
+Equipment,Chandelier,LightSource,Chandelier,,
+Equipment,Pendant,LightSource,Pendant,,
+Equipment,WallLight,LightSource,Wall Light,,
+Equipment,Sconce,LightSource,Sconce,,
+Equipment,AccentLight,LightSource,Accent Light,,
+Equipment,Lamp,LightSource,Lamp,,
+Equipment,FloodLight,LightSource,Flood Light,,
+Equipment,SpotLight,LightSource,Spot Light,,
+Equipment,WirelessAccessPoint,NetworkAppliance,Wireless Access Point,"AccessPoint, WAP, WiFi",
+Equipment,NetworkSwitch,NetworkAppliance,Network Switch,,
+Equipment,Router,NetworkAppliance,Router,,
+Equipment,Firewall,NetworkAppliance,Firewall,,
+Equipment,PetCare,,Pet Care,,
+Equipment,PetFeeder,PetCare,Pet Feeder,,
+Equipment,CatFlap,PetCare,Cat Flap,,
+Equipment,Aquarium,PetCare,Aquarium,,
+Equipment,AudioVisual,,Audio Visual,,
+Equipment,MediaPlayer,AudioVisual,Media Player,,
+Equipment,WaterFeature,Pump,Water Feature,"Water fall, Pond pump",
+Equipment,Printer,,Printer,,
+Equipment,Tool,,Tool,"Machine tool, CNC tool, 3D Printer",
```

Comment:
```suggestion
Equipment,Tool,,Tool,,
```
This is also a super generic equipment from which more specific ones can be derived. We don't want to add the synonyms here for they might clash with the sub-types.